src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/utils/UserReceiptMailBuilder.kt

```diff
@@ -109,8 +109,8 @@ class UserReceiptMailBuilder(@Autowired private val confidentialMailUtils: Confi
             PspTemplate(
               transactionAuthorizationRequestData.pspBusinessName,
               FeeTemplate(amountToHumanReadableString(transactionAuthorizationRequestData.fee))),
-            transactionAuthorizationRequestData
-              .authorizationRequestId, // FIXME: put here RRN received from PGS
+            Optional.ofNullable(transactionAuthorizationCompletedData.rrn)
+              .orElse(transactionAuthorizationRequestData.authorizationRequestId),
```

Comment:
```suggestion
            transactionAuthorizationCompletedData.rrn?:
              transactionAuthorizationRequestData.authorizationRequestId,
```
Replace with elvis operator